docs/source/getting-started.rst

```diff
@@ -0,0 +1,90 @@
+
+===============
+Getting started
+===============
+
+This getting started guide walks you through using Anaconda Project for the first time. 
+
+**After completing this guide, you will be able to:**
+
+* Create a project containing a Bokeh app
+* Package and share the project
+* Run the project with a single command
+
+This guide is for all platforms: Windows, OS X and Linux.
+
+If you have not yet installed and started Anaconda Project, follow the :doc:`Install instructions <install>`.
+
+NOTE: Windows users using Python 3.5.1 should upgrade to Python 3.5.2+ with the command ``conda update python`` to avoid an issue with Windows and Python 3.5.1.
```

Comment:
might be safe to remove this note now - someone could still encounter the problem (I don't even remember what it was) but doesn't seem like it will be common.